newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -297,275 +298,303 @@ export const GameDetailsDialog = ({
-            <HelpButton
-              key="help"
-              helpPagePath={
-                currentTab === 'leaderboards'
-                  ? '/interface/games-dashboard/leaderboard-administration'
-                  : '/interface/games-dashboard'
+    <ResponsiveWindowMeasurer>
+      {windowWidth => (
+        <I18n>
+          {({ i18n }) => (
+            <Dialog
+              title={
+                <span>
+                  {game.gameName}
+                  {' - '}
+                  <Trans>Dashboard</Trans>
+                </span>
               }
-            />,
-          ]}
-          onRequestClose={onClose}
-          cannotBeDismissed={isLoading}
-        >
-          <Tabs value={currentTab} onChange={setCurrentTab}>
-            <Tab label={<Trans>Details</Trans>} value="details" />
-            <Tab label={<Trans>Builds</Trans>} value="builds" />
-            <Tab label={<Trans>Feedback</Trans>} value="feedback" />
-            <Tab label={<Trans>Analytics</Trans>} value="analytics" />
-            <Tab label={<Trans>Leaderboards</Trans>} value="leaderboards" />
-          </Tabs>
-          <Line expand>
-            {currentTab === 'leaderboards' ? (
-              <LeaderboardAdmin gameId={game.id} onLoading={setIsLoading} />
-            ) : null}
-            {currentTab === 'details' ? (
-              publicGameError ? (
-                <PlaceholderError onRetry={loadPublicGame}>
-                  <Trans>There was an issue getting the game details.</Trans>{' '}
-                  <Trans>
-                    Verify your internet connection or try again later.
-                  </Trans>
-                </PlaceholderError>
-              ) : !publicGame ? (
-                <PlaceholderLoader />
-              ) : (
-                <ColumnStackLayout expand>
-                  {!isGameOpenedAsProject && (
-                    <AlertMessage kind="info">
+              open
+              noMargin
+              flexColumnBody
+              fullHeight={currentTab === 'leaderboards'}
+              maxWidth="md"
+              actions={[
+                <FlatButton
+                  label={<Trans>Close</Trans>}
+                  disabled={isLoading}
+                  onClick={onClose}
+                  key="close"
+                />,
+              ]}
+              secondaryActions={[
+                <HelpButton
+                  key="help"
+                  helpPagePath={
+                    currentTab === 'leaderboards'
+                      ? '/interface/games-dashboard/leaderboard-administration'
+                      : '/interface/games-dashboard'
+                  }
+                />,
+              ]}
+              onRequestClose={onClose}
+              cannotBeDismissed={isLoading}
+            >
+              <Tabs
+                value={currentTab}
+                onChange={setCurrentTab}
+                variant={windowWidth === 'medium' ? 'scrollable' : undefined}
```

Comment:
Tabs won't be scrollable when windowWidth === "small", is this normal?
If not, prefer to check if windowWidth !== "large"